beam-postgres/Database/Beam/Postgres/Full.hs

```diff
@@ -36,7 +36,10 @@ module Database.Beam.Postgres.Full
   , onConflictDefault, onConflict
   , conflictingConstraint
   , onConflictUpdateSet
-  , onConflictUpdateSetWhere, onConflictUpdateInstead
+  , onConflictUpdateSetWhere,
```

Comment:
`onConflictUpdateSet` and `onConflictUpdateSetWhere` are both already included in `BeamHasInsertOnConflict(..)`.

Sorry to keep going back and forth like this. :smile:

Reply:
Updated